tools/generate_hash_table.py

```diff
@@ -35,7 +35,7 @@ def build_hash_table(dbase_root):
 @click.command()
 @click.option('--database', required=True, type=str)
 def write_hash_table(database):
-    data_dir = pathlib.Path(get_pkg_data_path('data', package='fiasco.util'))
+    data_dir = pathlib.Path(get_pkg_data_path('data', package='fiasco.tests'))
```

Comment:
Wow, I don't know how that happened. Thanks for catching that!

Reply:
I only noticed by chance.  I was trying to build the hash files as a test and it spit out an error.  Simple enough fix!